airflow/config_templates/config.yml

```diff
@@ -453,6 +453,15 @@
       type: string
       example: ~
       default: "128"
+    - name: max_map_size
+      description: |
+        The maximum list/dict size an XCom can push to trigger task mapping. If the pushed list/dict
```

Comment:
Should we change `size` to `length` ? Currently it ambiguous and feels like it might be storage size in bytes/kbs etc

Reply:
Probably same for the name itself - `max_map_size` -> `max_map_length` / `max_map_count` -- or something else 🤷

Reply:
No strong preference from me. `length` is probably _marginally_ better (because `len`) but not really much difference. “Dict length” sounds weird to me conceptually.

Reply:
@BasPH @ephraimbuddy  Any thoughts/opinions ?

Reply:
Don't block this PR because of this -- I don't have a strong preference either and if others don't feel the same way too, feel free to merge it :)

Reply:
I prefer length but if we should use size, I think having a default of 1024 will make it seem like a storage size in kb

Reply:
+1 for length, size suggests a unit in bytes to me